### PR TITLE
feat: add isReversed prop to FilterMultiSelect

### PR DESCRIPTION
**Pull request by BrendanTimmons**
## Why
Need `isReversed` functionality from `Select` component to the `FilterMultiSelect` component because we're using both in a filters area on a purple background.

## What
Add the required `isReversed` functionality to `FilterMultiSelect`

### Default:
Screen Shot 2023-01-27 at 12 38 04 pm

### Reversed:
Screen Shot 2023-01-27 at 12 35 28 pm

### CODE REVIEW

File: packages/select/src/FilterMultiSelect/components/Trigger/TriggerButtonBase/TriggerButtonBase.module.scss [hunk 1]
```diff
@@ -12,3 +13,63 @@
...
+      color: $color-white;
+    }
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  #{$selectors--button-focus} {
+    background-color: rgba($color-white-rgb, 0.1);
```
Comment (NatalieLoyzaga) [inline packages/select/src/FilterMultiSelect/components/Trigger/TriggerButtonBase/TriggerButtonBase.module.scss:46]: The [UI kit](https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%EF%B8%8F%F0%9F%96%BC%EF%B8%8F-Component-Gallery?node-id=1206%3A155907&t=rVqn5LiNNw3fUPxn-0) has 0.2 opacity 

`background-color: rgba($color-white-rgb, 0.2);`

File: packages/select/src/FilterMultiSelect/components/Trigger/TriggerButtonBase/TriggerButtonBase.module.scss [hunk 2]
```diff
@@ -12,3 +13,63 @@
...
+        $color-blue-300;
+      border-radius: $border-focus-ring-border-radius;
+      top: calc(-1 * #{$focus-ring-offset});
+      left: calc(-1 * #{$focus-ring-offset});
+      right: calc(-1 * #{$focus-ring-offset});
+      bottom: calc(-1 * #{$focus-ring-offset});
+    }
+  }
+
+  .error {
```
Comment (NatalieLoyzaga) [inline packages/select/src/FilterMultiSelect/components/Trigger/TriggerButtonBase/TriggerButtonBase.module.scss:68]: We don't require the `.error` and `.caution` styles

File: packages/select/src/FilterMultiSelect/components/Trigger/_variables.scss [hunk 3]
```diff
@@ -8,3 +8,13 @@ $polyfill--focus-visible: ":global(.js-focus-visible) &:global(.focus-visible)";
...
+
+// Classnames to simulate pseudo states in storybook
+$story-className--button-hover: ":global(.story__button-hover)";
+$story-className--button-active: ":global(.story__button-active)";
+$story-className--button-focus: ":global(.story__button-focus)";
+
+// Combined pseudo state classes
+$selectors--button-hover: "&:hover, &#{$story-className--button-hover}";
+$selectors--button-active: "&:active, &#{$story-className--button-active}";
+$selectors--button-focus: "&:focus, &#{$story-className--button-focus}";
```
Comment (NatalieLoyzaga) [inline packages/select/src/FilterMultiSelect/components/Trigger/_variables.scss:20]: This is a double up of line 10

**Review by NatalieLoyzaga — CHANGES_REQUESTED** [review]
Hey @BrendanTimmons I just left a few comments :)

**Comment by BrendanTimmons** [comment]
Closing this as we went with a different solution and just used the filter multi select for all our selects instead of a mix of select / multiselects.

**Pull request closed without merging**